### DISCUSSION TITLE
Yield total and day: In AC or DC?

### DISCUSSION THREAD

**Discussion by MichaelPhilizot** [tbnobody/OpenDTU#1932] [General]
Hi

I was just wondering: the yieldday and yieldtotal values are DC or AC?

Another way of asking: is the yieldtotal the sum of what the microinverter injected in my electric network? Or should this value multiplicated by the efficiency to get the sum of what the microinverter injected in my electric network?

If anyone knows the answer, thanks in advance to share it.

Michaël

**Comment by Scootch2002** [comment]
AC only. DC makes no sense.
